src/globus_sdk/services/transfer/client.py

```diff
@@ -39,6 +36,25 @@ class TransferClient(client.BaseClient):
     :type authorizer: :class:`GlobusAuthorizer\
                       <globus_sdk.authorizers.base.GlobusAuthorizer>`
 
+    **Paginated Calls**
+
+    Methods which support pagination can be called as paginated or unpaginated methods.
+    If the method name is ``TransferClient.foo``, the paginated version is
+    ``TransferClient.paginated.foo``.
+    Using ``TransferClient.endpoint_search`` as an example::
+
+        from globus_sdk import TransferClient
+        tc = TransferClient(...)
+
+        # this is the unpaginated version
+        for x in tc.endpoint_search("tutorial"):
+            print("Endpoint ID: {}".format(x["id"]))
+
+        # this is the paginated version
```

Comment:
Should this example use .items() instead of a nested for loop? Or is that not our recommended usage?

Reply:
Yeah, let's make sure to come back to this.

I actually want to put together more work on the sphinx extension to automatically trawl methods with `_paginator_class` to generate more doc, so there's a whole pass on doc refinement we can make. I'll write it up in a story.